src/components/common.js

```diff
@@ -201,30 +201,49 @@ export const comingSoon = span({
 
 export const Select = RSelect
 
-export const LargeFadeBox = ({ children }) => {
+export const FadeBox = ({ fadePoint = '60%', style, children }) => {
+  const {
+    paddingTop = '1.5rem',
+    paddingLR = '1.5rem',
+    borderRadius = '8px',
+    backgroundColor = colors.gray[5],
+    borderColor = colors.gray[3],
+    ...containerStyle
+  } = style || {}
```

Comment:
TOL: It might be a little confusing that this is called `style`, but doesn't get treated exactly like a standard style object. Maybe we should rename it, or split it into standard vs nonstandard attributes?